app/src/main/java/com/toandv/demo/calculator/KeyboardFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.toandv.demo.calculator
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.core.os.bundleOf
+import androidx.core.view.children
+import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.fragment_keyboard.*
+
+class KeyboardFragment : Fragment(), View.OnClickListener {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_keyboard, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        for (item in constraintKeyboard.children) {
+            if (item is Button) item.setOnClickListener(this)
+        }
+    }
+
+    override fun onClick(p0: View?) {
```

Comment:
Đặt lại tên cho ý nghĩa hơn nhé 
```suggestion
    override fun onClick(view: View?) {
```